test/e2e/fixtures.go

```diff
@@ -130,42 +129,44 @@ func setupTest(tb testing.TB) (*TestData, error) {
 }
 
 func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
-	data := &TestData{}
+	// TODO: remove hardcoding to IPv4 after flow aggregator supports IPv6
 	isIPv6 := false
-	if err := data.setupLogDirectoryForTest(tb.Name()); err != nil {
-		tb.Errorf("Error creating logs directory '%s': %v", data.logsDirForTestCase, err)
+	if err := testData.setupLogDirectoryForTest(tb.Name()); err != nil {
```

Comment:
idea for later: this should just call `setupTest` instead of duplicating the code

Reply:
Yes added in above TODO